### PR TITLE
Use print() function in both Python 2 and Python 3

### PR DESCRIPTION
**Pull request by cclauss**
Legacy __print__ statements are syntax errors in Python 3 but __print()__ function works as expected in both Python 2 and Python 3.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @cclauss! I've created [OSPR-2973](https://openedx.atlassian.net/browse/OSPR-2973) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

**Comment by natabene** [comment]
@cclauss  Thank you for the contribution! When you have a chance, could you please send in a signed Contributor Agreement mentioned in the earlier comment? You only need to do this once and we can then review all your future code contributions.

**Comment by natabene** [comment]
@nedbat Can you have a look?

File: pavelib/paver_tests/utils.py [hunk 1]
```diff
@@ -4,6 +4,11 @@
 from paver import tasks
 from unittest import TestCase
 
+try:
+    unicode        # Python 2
+except NameError:  # Python 3
+    unicode = str  # pylint: disable=invalid-name,redefined-builtin
+
```
Comment (nedbat) [inline pavelib/paver_tests/utils.py]: Per [OEP-7](https://open-edx-proposals.readthedocs.io/en/latest/oep-0007-bp-migrate-to-python3.html), we would rather use six than use ad-hoc compatibility code like this.

Comment (cclauss) [reply pavelib/paver_tests/utils.py]: Done.

**Comment by natabene** [comment]
@cclauss  Thank you, we can continue review now. CC @nedbat

**Comment by cclauss** [comment]
Bump on this one...  I would like to take a shot at https://openedx.atlassian.net/browse/INCR-83 but need these basic issues cleared before I do.

**Comment by natabene** [comment]
@edx/incr-reviewers Can you give this a look?

**Comment by edx-webhook** [comment]
@cclauss 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-e2e-tests&entry.867055334=https://github.com/edx/edx-e2e-tests/pull/370&entry.1484655318=https://github.com/cclauss&entry.752974735=2019-01-13+16:42&entry.1917517419=2019-02-23+20:44&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**